[2023-06-09T20:00:37.436+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: yellow_taxi_gcs_dag.download_dataset_task scheduled__2019-07-02T06:00:00+00:00 [queued]>
[2023-06-09T20:00:37.443+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: yellow_taxi_gcs_dag.download_dataset_task scheduled__2019-07-02T06:00:00+00:00 [queued]>
[2023-06-09T20:00:37.444+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-06-09T20:00:37.444+0000] {taskinstance.py:1363} INFO - Starting attempt 2 of 2
[2023-06-09T20:00:37.445+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-06-09T20:00:37.453+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-07-02 06:00:00+00:00
[2023-06-09T20:00:37.457+0000] {standard_task_runner.py:54} INFO - Started process 1080 to run task
[2023-06-09T20:00:37.461+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'yellow_taxi_gcs_dag', 'download_dataset_task', 'scheduled__2019-07-02T06:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_multiFile.py', '--cfg-path', '/tmp/tmpj53l5adm']
[2023-06-09T20:00:37.462+0000] {standard_task_runner.py:83} INFO - Job 19: Subtask download_dataset_task
[2023-06-09T20:00:37.463+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/data_ingestion_gcs_multiFile.py
[2023-06-09T20:00:37.561+0000] {data_ingestion_gcs_multiFile.py:27} INFO - Dataset url is https://github.com/DataTalksClub/nyc-tlc-data/releases/download/yellow/yellow_tripdata_{{ execution_date.strftime('%Y-%m') }}.csv.gz, local_home is /opt/***, parquet file is /opt/***yellow_tripdata_{{ execution_date.strftime('%Y-%m') }}.parquet
[2023-06-09T20:00:37.564+0000] {data_ingestion_gcs_multiFile.py:35} INFO - dtc_data_lake_dtc-dezc-001 is the bucket.
[2023-06-09T20:00:37.598+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-06-09T20:00:37.649+0000] {task_command.py:384} INFO - Running <TaskInstance: yellow_taxi_gcs_dag.download_dataset_task scheduled__2019-07-02T06:00:00+00:00 [running]> on host 3a78d9143ce8
[2023-06-09T20:00:37.677+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-06-09T20:00:37.715+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=yellow_taxi_gcs_dag
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-07-02T06:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-07-02T06:00:00+00:00
[2023-06-09T20:00:37.717+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-09T20:00:37.717+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'curl -sSLf https://github.com/DataTalksClub/nyc-tlc-data/releases/download/yellow/yellow_tripdata_2019-07.csv.gz > /opt/***yellow_tripdata_2019-07.csv.gz']
[2023-06-09T20:00:37.726+0000] {subprocess.py:86} INFO - Output:
[2023-06-09T20:00:37.729+0000] {subprocess.py:93} INFO - /bin/bash: line 1: /opt/***yellow_tripdata_2019-07.csv.gz: Permission denied
[2023-06-09T20:00:37.730+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-06-09T20:00:37.742+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-06-09T20:00:37.745+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=yellow_taxi_gcs_dag, task_id=download_dataset_task, execution_date=20190702T060000, start_date=20230609T200037, end_date=20230609T200037
[2023-06-09T20:00:37.757+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 19 for task download_dataset_task (Bash command failed. The command returned a non-zero exit code 1.; 1080)
[2023-06-09T20:00:37.795+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-06-09T20:00:37.816+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
